README.md

```diff
@@ -252,7 +252,8 @@ objectives:
   - displayName: string # optional
     op: lte | gte | lt | gt # conditional operator used to compare the SLI against the value. Only needed when using a thresholdMetric
     value: numeric # optional, value used to compare threshold metrics. Only needed when using a thresholdMetric
-    target: numeric [0.0, 1.0) # budget target for given objective of the SLO
+    target: numeric [0.0, 1.0) # budget target for given objective of the SLO, can't be used with targetPercent
+    targetPercent: numeric [0.0, 100) with optional "%" suffix # budget target for given objective of the SLO, can't be used with target
```

Comment:
Do we need the "optional "%" suffix if the word "percent" is in the name?

Reply:
Yeah, doesn't look to pretty. Dropped it.